source/base/job_identifier.cc

```diff
@@ -20,7 +20,7 @@
 # include <unistd.h>
 #endif
 
-#ifdef DEAM_II_MSVC
+#ifdef DEAL_II_MSVC
 #  include <WinSock2.h>
```

Comment:
So I suppose you just found this by luck and personal inspection, rather than realizing that there was a compiler error in this file that required `WinSock2.h`.